### PR TITLE
Add WingIDE.app v5.1.8-1

### PR DESCRIPTION
**Pull request by vman747**
Trial version of Pro
Requires key to unlock after 30 days no new downloads required

### CODE REVIEW

**Comment by vitorgalvao** [comment]
If it’s a timed trial, it is `:commercial`, not `:freemium`.

File: Casks/wingide.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'wingide' do
+  version '5.1.8-1'
+  sha256 'e3f12e467fd0a17160406010cc6d6fd2b7c84e725848cce4694307706a8eb6c8'
+
+  url 'http://wingware.com/pub/wingide/5.1.8/wingide-5.1.8-1.dmg'
```
Comment (vitorgalvao) [inline Casks/wingide.rb]: `url "http://wingware.com/pub/wingide/#{version}/wingide-#{version}.dmg"`

Comment (adityadalal924) [inline Casks/wingide.rb]: The first version doesn't have the `-1` appended, so needs the appropriate regex.

Comment (vitorgalvao) [inline Casks/wingide.rb]: Nice catch, you’re right. Fixed.

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 4d242fd14dc9578d30fb89f2653d84974285d692. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**